server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/AbstractStringTermsAggregator.java

```diff
@@ -33,7 +33,7 @@
 
 import static java.util.Collections.emptyList;
 
-abstract class AbstractStringTermsAggregator extends TermsAggregator {
+public abstract class AbstractStringTermsAggregator extends TermsAggregator {
```

Comment:
Why is this needed? I don't see this class being use anywhere extra but maybe I'm missing something?